life_hc/MySQL/General/fn_insertVehicle.sqf

```diff
@@ -13,13 +13,14 @@ params [
     "_side",
     "_type",
     "_className",
+    "_shop",
```

Comment:
it's a string, doesn't need a default value. It'll always be "".
https://community.bistudio.com/wiki/params